libgalois/src/analytics/independent_set/independent_set.cpp

```diff
@@ -17,52 +17,43 @@
  * Documentation, or loss or inaccuracy of data of any kind.
  */
 
-#include <math.h>
+#include "katana/analytics/independent_set/independent_set.h"
```

Comment:
The style guide says this should be first because we are implementing it in this file.

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
ohh got it. My bad.